routes.py

```diff
@@ -379,4 +397,22 @@ async def delete_meeting(id: str):
     if delete_result.deleted_count == 1:
         return Response(status_code=status.HTTP_204_NO_CONTENT)
 
-    raise HTTPException(status_code=404, detail=f"meeting {id} not found")
\ No newline at end of file
+    raise HTTPException(status_code=404, detail=f"meeting {id} not found")
+
+
+# ********** Utils **********
+
+async def get_user(user_id: str) -> dict:
+    user_found = await user_collection.find_one({"_id": ObjectId(user_id)})
+    if user_found is None:
+        raise HTTPException(status_code=404, detail=f"user {user_id} not found")
+    return user_found
+
+
+async def update_array_element():
+    pass
+
+
+if __name__ == "__main__":
```

Comment:
What's this for?

Reply:
Used it to run the API on the local host, instead of the docker container to debug it
removed